dask_labextension/manager.py

```diff
@@ -23,8 +23,12 @@
 async def make_cluster(configuration: dict) -> Cluster:
     module = importlib.import_module(dask.config.get('labextension.factory.module'))
     Cluster = getattr(module, dask.config.get('labextension.factory.class'))
+
+    kwargs = dask.config.get('labextension.factory.kwargs').copy()
+    if 'n_workers' not in kwargs and utils.has_keyword(Cluster, 'n_workers'):
```

Comment:
This seems to make an assumption about the keyword arguments of the factory function that we had not been making before, which was quite general. Do you ask that all cluster constructors implement `n_workers`? Might this be fragile if `n_workers` does something different than we expect?